src/wolfgame.py

```diff
@@ -2440,7 +2440,9 @@ def stop_game(cli, winner = "", abort = False, additional_winners = None):
 
     if not abort:
         cli.msg(chan, gameend_msg)
-
+        # Message Players in Deadchat letting them know that the game has ended. cyberfawkes
+        mass_privmsg(cli, var.DEADCHAT_PLAYERS, messages["endgame_deadchat"].format(chan))
```

Comment:
This should be moved to after the end-of-game stuff has been run, but before reset(). Displaying the list of winners and recording the stats is more important than letting the people in deadchat know the game is over. Also, remove your name from the comment. Nobody "owns" the code, and git already supplies attribution information.